docker-compose.yaml

```diff
@@ -6,15 +6,21 @@ services:
     container_name: wdio-cucumber
     links:
```

Comment:
`links` is deprecated and will be removed in future.  It doesn't really offer any benefit in terms of access as all containers can talk to all ports on all other containers in the same docker network, it's just to help with starting up.

Suggest you change `links` to `depends_on` to keep the containers starting up in the right order.

Reply:
Right, understood